shared/main/java/net/ccbluex/liquidbounce/features/module/modules/render/XRayBypass.java

```diff
@@ -0,0 +1,42 @@
+package net.ccbluex.liquidbounce.features.module.modules.render;
+
+import net.ccbluex.liquidbounce.features.module.Module;
+import net.ccbluex.liquidbounce.features.module.ModuleCategory;
+import net.ccbluex.liquidbounce.features.module.ModuleInfo;
+import net.minecraft.block.Block;
+import net.minecraft.client.Minecraft;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+
+@ModuleInfo(name = "XRayBypass", description = "", category = ModuleCategory.RENDER)
+public class XRayBypass extends Module {
+
+    private static final int i = 16;
+    private int retardedInt;
+    private int retardedInt2;
+
+    private int retardedInt3;
```

Comment:
Rename fields to something more descriptive of what their purpose is, so e.g. `xPos` instead of `retardedInt3` etc.